bpython/test/test_autocomplete.py

```diff
@@ -365,3 +367,6 @@ def func(apple, apricot, banana, carrot):
                             set(['banana=']))
         self.assertSetEqual(com.matches(3, "car", argspec=argspec),
                             set(['carrot=']))
+
```

Comment:
Instead of this, you can run individual tests with `nose tests/test_autocomplete.py`